### PR TITLE
Add SDXL EulerDiscreteScheduler compilation test

### PR DESCRIPTION
**Pull request by saienduri**
This commit adds a compilation test for the scheduler used in sdxl. We can iterate on this and add some tests that runs the different functions as well (`run_initialize`, `run_scale`, `run_step`) and checks for accuracy.

ci-exactly: build_packages, regression_test

### CODE REVIEW

File: experimental/regression_suite/shark-test-suite-models/sdxl/test_scheduler.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers_tools import *
+import os
+from conftest import VmfbManager
+from pathlib import Path
+
+rocm_chip = os.getenv("ROCM_CHIP", default="gfx90a")
```
Comment (raikonenfnu) [inline experimental/regression_suite/shark-test-suite-models/sdxl/test_scheduler.py]: Is `ROCM_CHIP` always set in the builders? I think in test_unet.py is defaulted to gfx942, is there any reason we use gfx90a instead?

**Review by raikonenfnu — COMMENTED** [review]
Overall looks good, just a QQ

File: experimental/regression_suite/shark-test-suite-models/sdxl/test_scheduler.py (hunk 1 above)
Comment (saienduri) [reply experimental/regression_suite/shark-test-suite-models/sdxl/test_scheduler.py]: Yeah it's always set as part of the workflow file, so runners always get gfx942. This default is more for end users. I just changed all defaults to gfx942, so it's consistent though

**Review by raikonenfnu — APPROVED** [review]

**Comment by raikonenfnu** [comment]
Nice, thanks for clean up :)

**Pull request merged**